examples/python/saved_models/README.md

```diff
@@ -0,0 +1 @@
+# Directory for saved models
```

Comment:
This directory could be added with `os.makedirs(path)` rather than being included in git, or alternatively just save the model in the same directory as the example file (it is an example, so does not have to be the cleanest solution ever).

Reply:
Done